### ISSUE TITLE
Logo For Diffsims

### ISSUE THREAD

**Issue by CSSFrancis** [pyxem/diffsims#192]
This is something that has come up a couple of times when trying to explain the pyxem hierarchy.  I think that diffsims needs a logo.  I've starting writing a pyxem paper and was working on a dependency tree figure that doesn't really work without a logo.

I can draw up some designs. I'm not sure if @pc494 you guys had a logo when it was first created? 

I was thinking something like this:

<img width="332" alt="image" src="https://user-images.githubusercontent.com/41125831/232075274-ad163403-1122-4579-a009-1dbf902f9ba5.png">

I need to tweak it a little bit and make sure the colors match the pyxem logo but I figure it might be good to have a constant theme.

**Comment by hakonanes** [comment]
Great initiative. Sorry to spoil your particular effort here a bit, but the logo should reflect diffraction, not diffraction in TEM. diffsims, particularly `diffsims.crystallography.ReciprocalLatticeVector`, is much used in kikuchipy (see e.g. our [kinematical simulations tutorial](https://kikuchipy.org/en/stable/tutorials/kinematical_ebsd_simulations.html)), and it would be unfortunate if the logo does not reflect diffraction in the SEM as well.

I'm afraid I don't have a suggestion for a logo myself... Is it possible to use the pyxem logo for the time being?

**Comment by CSSFrancis** [comment]
Fair enough!  My original idea was to using something like a more classic diffraction example but worried that it didn't really describe TEM diffraction very well.  I fear that I may have gone too far in the opposite direction.  

We could just have something like this?

![](http://pd.chem.ucl.ac.uk/pdnn/diff2/wavedif.gif)

Or combine the two?

**Comment by CSSFrancis** [comment]
Otherwise we could just take the pyxem logo and replace the text with "diffsims"

**Comment by CSSFrancis** [comment]
Just playing around with it....

I probably need to work on the realism of my Kikuchi pattern and we could add in an example of X-ray diffraction as well. 


<img width="484" alt="image" src="https://user-images.githubusercontent.com/41125831/232083678-70ca9bb6-e5d5-4557-bfe0-fc4338915b61.png">

**Comment by hakonanes** [comment]
I like your idea of having a probe, a material with some property of interest, and a product out. We use diffsims in our forward models to predict the outcome of an experiment. Would be good if the logo captures this somehow, with relation to diffraction.

**Comment by pc494** [comment]
We never had a diffsims logo. My suggestions would be:

1) simplify a lot, it needs to vaguely make sense when very small on a big screen
2) I wouldn't use the pyxem colour scheme, will struggle to make them distinct, could use black as the background?
3) I also don't really like text on logos (can you guess that the pyxem logo wasn't my design).

In the end it's a logo, it just needs to vaguely look sensible, it doesn't need to capture all that the package can do. Even three planes, and a stylised wave on a black background would do it for me.

**Comment by hakonanes** [comment]
+1 for the planes and wave.

**Comment by CSSFrancis** [comment]
So something like this?

<img width="480" alt="image" src="https://user-images.githubusercontent.com/41125831/232340009-518795c1-2352-41c5-ac49-e134981eb36f.png">

I can play around with the colors a bit.  Right now it might be too close to the background pyxem color.

**Comment by pc494** [comment]
I really like this @CSSFrancis

**Comment by CSSFrancis** [comment]
@pc494 perfect I'll vectorize the image and create a svg. Admittedly this was my first concept but then I kind of overthought it. :)

**Comment by hakonanes** [comment]
Looks great! To make it clear when smaller, say, as a favicon in the browser in the extreme case, is it possible to increase the wave amplitude and the line widths?

**Comment by hakonanes** [comment]
I suddenly got a musical sheet vibe when looking at the logo (:

**Comment by CSSFrancis** [comment]
Just playing around with some line thicknesses... 

![diffsimslogo](https://user-images.githubusercontent.com/41125831/232881742-76f92216-c2ae-4121-9619-3f49dacfc918.svg)

![diffsims](https://user-images.githubusercontent.com/41125831/232881762-702070ee-8db5-464c-b7cb-82cfffc19474.svg)

![diffsims2](https://user-images.githubusercontent.com/41125831/232881668-6f163886-8046-42f7-8eb0-5ac229135502.svg)
